torch/_dynamo/utils.py

```diff
@@ -195,7 +204,7 @@ def print_time_report():
 # The frame is incremented outside of this function, in def increment_frame() above.
 
 
-def dynamo_timed(original_function=None, phase_name=None):
+def dynamo_timed(original_function=None, phase_name=None, fwd_only=True):
```

Comment:
I couldn't tell from the use sites what this kwarg does, some docs would be appreciated

Reply:
Will add docs!